csrc/quantization/gptq_marlin/marlin_template.h

```diff
@@ -681,7 +690,7 @@ __global__ void Marlin(
     sh_first_group_id = first_group_id;
     sh_num_groups = last_group_id - first_group_id + 1;
 
-    if (sh_num_groups < act_s_max_num_groups) {
+    if (sh_num_groups > act_s_max_num_groups) {
```

Comment:
Was this a bug for act order that we didn't catch? Why was it found in this work

Reply:
Some test cases was failed on specail devices (since the out-of-range scales have overwritten sh_a). I'm not sure why this issue didn't occur in the previous PR.